chips/src/main/java/com/decathlon/vitamin/compose/chips/VitaminChipColors.kt

```diff
@@ -0,0 +1,213 @@
+
+    private val selectedBackgroundColor: Color,
+    private val selectedContentColor: Color,
+    private val selectedBorderColor: Color,
+
+    private val disabledSelectedBackgroundColor: Color,
+    private val disabledSelectedContentColor: Color,
+    private val disabledSelectedBorderColor: Color
+) : VitaminSelectableChipColors {
+
+    @Composable
+    override fun backgroundColor(enabled: Boolean, selected: Boolean): State<Color> {
+        return animateColorAsState(
+            when {
+                enabled && !selected -> backgroundColor
+                enabled && selected -> selectedBackgroundColor
+                !selected -> disabledBackgroundColor
+                else -> disabledSelectedBackgroundColor
+            }
+        )
+    }
+
+    @Composable
+    override fun contentColor(enabled: Boolean, selected: Boolean): State<Color> {
+        return animateColorAsState(
+            when {
+                enabled && !selected -> contentColor
+                enabled && selected -> selectedContentColor
+                !selected -> disabledContentColor
+                else -> disabledSelectedContentColor
+            }
+        )
+    }
+
+    @Composable
+    override fun leadingIconColor(enabled: Boolean, selected: Boolean): State<Color> {
+        return contentColor(enabled = enabled, selected = selected)
+    }
+
+    @Composable
+    override fun borderColor(enabled: Boolean, selected: Boolean): State<Color> {
+        return animateColorAsState(
+            when {
+                enabled && !selected -> borderColor
+                enabled && selected -> selectedBorderColor
+                !selected -> disabledBorderColor
+                else -> disabledSelectedBorderColor
+            }
+        )
+    }
+}
+
+@Immutable
+internal class VitaminDefaultChipColors(
+    private val backgroundColor: Color,
+    private val contentColor: Color,
+    private val borderColor: Color,
+
+    private val disabledBackgroundColor: Color,
+    private val disabledContentColor: Color,
+    private val disabledBorderColor: Color
+) : VitaminChipColors {
+    @Composable
+    override fun contentColor(enabled: Boolean): State<Color> {
+        return animateColorAsState(
+            if (enabled) contentColor else disabledContentColor
+        )
+    }
+
+    @Composable
+    override fun backgroundColor(enabled: Boolean): State<Color> {
+        return animateColorAsState(
+            if (enabled) backgroundColor else disabledBackgroundColor
+        )
+    }
+
+    @Composable
+    override fun leadingIconContentColor(enabled: Boolean): State<Color> {
+        return animateColorAsState(
+            if (enabled) contentColor else disabledContentColor
+        )
+    }
+
+    @Composable
+    override fun borderColor(enabled: Boolean): State<Color> {
+        return animateColorAsState(
+            if (enabled) borderColor else disabledBorderColor
+        )
+    }
+}
+
+object VitaminChipDefaults {
```

Comment:
I know why you named this class `Defaults` but because all other components use `Colors`, it should be the same too here.

Reply:
(Think to update documentation with this change)

Reply:
I will :
Change interfaces names 
Keep VitaminChipColors for public object